addons/product/views/product_views.xml

```diff
@@ -125,12 +125,14 @@
                                     </div>
                                     <label
                                         for="standard_price"
-                                        invisible="(product_variant_count &gt; 1 and not is_product_variant) or type == 'combo'"
+                                        invisible="((product_variant_count &gt; 1 or is_dynamically_created)
```

Comment:
```suggestion
                                        invisible="((product_variant_count &gt; 1 or (product_variant_count == 0 and is_dynamically_created))
```
It's confusing to include `product_variant_count == 0` in `is_dynamically_created`, the field content should match the field name.